lib/install.js

```diff
@@ -213,7 +215,31 @@ function install(opts, cb) {
     );
     fs.createReadStream(gzipFilePath).pipe(gunzip).pipe(writeStream)
       .once('error', cb.bind(null, new Error('Could not read ' + gzipFilePath)))
-      .once('finish', function() { cb() })
+      .once('finish', function() {
+        if (from.substr(-7) === '.tar.gz') {
+          var destDir = path.join(os.tmpdir(), 'selenium-standalone-' + Math.random());
```

Comment:
Probably, I will check for another module... Should be a good idea thought as it's a one file tar archive.
I make a commit this afternoon to correct it